Dockerfile

```diff
@@ -1,10 +1,27 @@
+# Container image that runs your code
 FROM registry.access.redhat.com/ubi8/ubi
 
 ARG conftest_version=0.30.0
+ARG BATS_VERSION=1.6.0
+
+ENV POLICY_PATH="/project"
+ENV JSON_PATH="${POLICY_PATH}/image_metadata.json"
 
 RUN curl -L https://github.com/open-policy-agent/conftest/releases/download/v"${conftest_version}"/conftest_"${conftest_version}"_Linux_x86_64.tar.gz | tar -xz --no-same-owner -C /usr/bin/ && \
     dnf -y --setopt=tsflags=nodocs install \
     jq \
     skopeo
 
-COPY policies /project
\ No newline at end of file
+# Installing BATS
+RUN curl -LO "https://github.com/bats-core/bats-core/archive/refs/tags/v$BATS_VERSION.tar.gz" && \
+	tar -xf "v$BATS_VERSION.tar.gz" && \
+	cd "bats-core-$BATS_VERSION" && \
+	./install.sh /usr && \
+	rm -rf "bats-core-$BATS_VERSION" && \
+        rm -rf "v$BATS_VERSION.tar.gz"
+
+COPY policies $POLICY_PATH
+COPY conftest.sh $POLICY_PATH
+COPY runtests.sh /runtests.sh
+
+ENTRYPOINT ["/runtests.sh"]
```

Comment:
Not sure if we should make the default entrypoint be the `runtests.sh` script

Reply:
I agree, I don't think we want the default entrypoint to be the runtests.sh script.